server/norace_test.go

```diff
@@ -3561,6 +3561,9 @@ func TestNoRaceJetStreamClusterCorruptWAL(t *testing.T) {
 		t.Fatalf("Error looking up consumer %q", "dlc")
 	}
 	// Grab underlying raft node and the WAL (filestore) and we will attempt to "corrupt" it.
+	o.mu.Lock()
+	o.skipSnapshotOnExit = true
```

Comment:
I think I would rather pull the consumer in this test and stop the raft node directly versus having a boolean for every consumer in normal operations.

So just call o.raftNode().Stop()

Reply:
Removed the boolean and did just that. Force pushed this commit.